### PR TITLE
Automated OCP-19675

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei please help to review, thanks

### CODE REVIEW

File: lib/rules/web/admin_console/4.4/check_links.xyaml [hunk 1]
```diff
@@ -161,3 +161,44 @@ check_customized_oc_download_links:
     text: 下载oc命令行
     link_url: mirror.openshift.com/pub/openshift-v4/clients/oc
   action: check_link_and_text
+
+# --- links on navigator menu ---
+check_Administration_menu_for_normal_user:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.4/check_links.xyaml]: `check_administration_menu_for_normal_user` is ok

Comment (yapei) [inline lib/rules/web/admin_console/4.4/check_links.xyaml]: `check_administration_menu_for_normal_user` is ok, it's better if we can keep consistence with other rules although `check_Administration_menu_for_normal_user` looks good

**Comment by SSshahan** [comment]
@pruan-rht  @liangxia   please help to merge, thanks

File: lib/rules/web/admin_console/4.3/check_links.xyaml [hunk 2]
```diff
@@ -165,3 +165,43 @@ check_customized_oc_download_links:
     text: 下载oc命令行
     link_url: mirror.openshift.com/pub/openshift-v4/clients/oc
   action: check_link_and_text
+
+# --- links on navigator menu ---
+check_Administration_menu_for_normal_user:
```
Comment (liangxia) [inline lib/rules/web/admin_console/4.3/check_links.xyaml]: Mix use of both upper and lower cases letters. Is it expected ? @SSshahan  @yapei

Comment (SSshahan) [reply lib/rules/web/admin_console/4.3/check_links.xyaml]: let me update

Comment (SSshahan) [reply lib/rules/web/admin_console/4.3/check_links.xyaml]: @liangxia  I have updated  the file and related to feature file. https://github.com/openshift/cucushift/pull/7485 please help to merge

**Comment by liangxia** [comment]
The CI issue is resolved via https://github.com/openshift/verification-tests/pull/831
/approve

**Pull request merged**